desktop/app/kbfs-helper.js

```diff
@@ -1,10 +1,32 @@
 // @flow
 
 import {ipcMain, shell} from 'electron'
+import fs from 'fs'
+import {pathToURL} from './paths'
 
 export default function () {
   ipcMain.on('openInKBFS', (e, path) => {
-    // We do the `shell.openItem` here to not block the ui thread
-    shell.openItem(path)
+    fs.stat(path, (err, stats) => {
```

Comment:
The hanging is unrelated to this PR (I think, unless some weird interplay is occurring from the `fs` commands). I've opened up a PR to log what I've seen/tried at https://github.com/keybase/client/pull/3737.